airbyte-webapp/src/components/ui/InfoBox/InfoBox.tsx

```diff
@@ -5,10 +5,11 @@ import styled from "styled-components";
 interface Props {
   className?: string;
   icon?: IconDefinition;
+  error?: boolean;
 }
 
-const Box = styled.div`
-  background: ${({ theme }) => theme.yellow100};
+const Box = styled.div<{ error?: boolean }>`
+  background: ${({ theme, error }) => (error ? theme.red50 : theme.yellow100)};
```

Comment:
Kind of a nitpick here: could we do something like `variant: 'default' | 'error'` instead of `error?: boolean`? That would make it easier later to add other variants of the `InfoBox`. It also avoids a potential situation like `<InfoBox error={true} someOtherStyleVariant={true} />` where the styling becomes unclear.

The prop signature for `InfoBox` could then set a default value like this: 
```
React.FC<React.PropsWithChildren<Props>> = ({ children, className, icon, variant = 'default' })
```

Reply:
I like it!